.github/workflows/release.yml

```
@@ -0,0 +1,113 @@
+                    | base64 -w0 \
+                    > check.sha256
+
+            - name: Upload Checksums
+              id: hashes
+              uses: slsa-framework/slsa-github-generator/actions/generator/generic/create-base64-subjects-from-file@f7dd8c54c2067bafc12ca7a55595d5ee9b75204a # 2.1.0
+              with:
+                  path: release-assets/check.sha256
+
+    AssetProvenance:
+        permissions:
+            actions:  read  # Needed for detection of GitHub Actions environment.
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)